src/app/asset-page/asset-page.component.ts

```diff
@@ -456,7 +456,7 @@ export class AssetPage implements OnInit, OnDestroy {
   } // OnInit
 
   get showDownloadDropdown() {
-    const isLoggedIn = this.user && this.user.isLoggedIn;
+    const isLoggedIn = (this.user && this.user.isLoggedIn) ? true : false;
```

Comment:
Pretty sure `!!(this.user && this.user.isLoggedIn)` will work

Reply:
I get that this is shorter syntax but it's not as easy to read 🤔